frontend/src/containers/Ledger.js

```diff
@@ -138,6 +150,15 @@ export class Ledger extends Component {
 
         { this.props.route.type === 'expenses' && <div className='Ledger-container padding40 expenses-container'>
             <div className='line1'>unpaid expenses</div>
+
+            {isHost && 
```

Comment:
shouldn't this be `isHost && !paypalCard` ?
I'd rather have that logic outside of the `<PaypalReminder />` component